tests/test_add_context.py

```diff
@@ -66,15 +66,17 @@ def setUp(self):
             os.remove(curr_file)
 
     def test_add_context(self):
```

Comment:
Let's add another test that asserts that if `add_context` is False, then it doesn't add those `otel...` fields.